pushnotifications-integration-tests/src/androidTest/java/com/example/pushnotificationsintegrationtests/FakeErrol.kt

```diff
@@ -49,19 +53,19 @@ class FakeErrol(port: Int, private val clusterKey: String = ""): NanoHTTPDRouter
     post("/instances/{instanceId}/devices/fcm") {
       entity(RegisterDeviceRequest::class) { registerDeviceRequest ->
         val device = FakeErrolDevice.New(registerDeviceRequest.token, mutableSetOf())
-        storage.devices[device.id] = device
+        getInstanceStorage(params["instanceId"]!!).devices[device.id] = device
```

Comment:
"instanceId" should be a const to be used by all the tests.

Reply:
This is the server, it makes no assumptions on instance ids. In fact, the tests use different (and multiple) instance ids.

Reply:
Oh I just meant the hardcoded string should be a const? we use the same hardcoded string 11 more times in this file.

Reply:
Ah, I see. So this string must match what's on the path. I guess you could interpolate in the path, and have it as a constant, but I don't think it really improves things that much in this case.

Reply:
Ah, I see. So this string must match what's on the path. I guess you could interpolate in the path, and have it as a constant, but I don't think it really improves things that much in this case.